### DISCUSSION TITLE
Google Chat Integration

### DISCUSSION THREAD

**Discussion by jorge-gbs** [outline/outline#4825] [Ideas]
Has adding a Google Chat integration been evaluated? It can just mirror the Slack integration. Especially since Google started supporting  slash commands last year.

**Comment by tommoor** [comment]
Honestly it's never really been asked for before, I get the impression there aren't many teams using Google Chat as their primary communication channel. But given we have Google SSO it makes total sense for it to be an option

**Comment by jochenstu** [comment]
Well, in that case let me +1 this :)
